src/include/rados.h

```diff
@@ -464,6 +464,19 @@ enum {
 
 const char *ceph_osd_watch_op_name(int o);
 
+enum {
+	CEPH_OSD_ALLOC_HINT_FLAG_SEQUENTIAL_WRITE = 1,
+	CEPH_OSD_ALLOC_HINT_FLAG_RANDOM_WRITE = 2,
```

Comment:
Any rationale to have specific flags for random write/read? Don't they duplicate corresponding sequential read/write flags set to 0?